README.md

```diff
@@ -139,6 +139,16 @@ On iOS installation is a bit more complicated, because you need to setup a scree
      pod 'MembraneRTC/Broadcast'
    end
    ```
+   > This new dependency should be added outside of your application target. Example
+   > ```rb
+   > target 'ReactNativeMembraneExample' do
+   >  ...
+   > end
+   > 
+   > target 'MembraneScreenBroadcastExtension' do
+   >  pod 'MembraneRTC/Broadcast'
+   > end
+   > ```
```

Comment:
Weird, it's already merged and it showed here, did you start with updated master branch? There were a lot of changes in the last commit.

Reply:
Yes, I forked the project and after I pull the latest changes from master, that's really weird